.vib/wordpress/cypress/cypress/integration/wordpress_spec.js

```diff
@@ -98,27 +98,30 @@ it('checks if admin can create a post', () => {
   cy.get('.editor-post-saved-state').and('have.text', 'Saved');
 });
 
-it('checks the SMTP configuration', () => {
-  cy.login();
-  cy.visit('/wp-admin/admin.php?page=wp-mail-smtp');
-  cy.contains('div', 'WP Mail SMTP');
-  cy.get('#wp-mail-smtp-setting-smtp-host').should(
-    'have.value',
-    Cypress.env('smtpMailServer')
-  );
-  cy.get('#wp-mail-smtp-setting-smtp-port').should(
-    'have.value',
-    Cypress.env('smtpPort')
-  );
-  cy.get('#wp-mail-smtp-setting-smtp-user').should(
-    'have.value',
-    Cypress.env('smtpUser')
-  );
-  cy.get('#wp-mail-smtp-setting-from_name').should(
-    'have.value',
-    `${Cypress.env('wordpressFirstName')} ${Cypress.env('wordpressLastName')}`
-  );
-});
+/* TEMPORARILY SHUTTING DOWN THIS TEST DUE TO MAINTENANCE  */
+
+// it('checks the SMTP configuration', () => {
```

Comment:
@superaleks I believe adding an `x` suffix disables the test without having to comment it all (dead code shall not pass!).
I knew it from some JS testing frameworks but it seems cypress also supports it:
https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Pending

Reply:
This is true, I changed the test to have `skip` in front of it, so now this test shouldn't execute until we make changes.
Thanks, I learned something new today!